SlicerCART/src/SlicerCART.py

```diff
@@ -2413,6 +2499,9 @@ def onEditConfiguration(self):
   def onSelectVolumesFolderButton(self):
       self.CurrentFolder= qt.QFileDialog.getExistingDirectory(None,"Open a folder", self.DefaultDir, qt.QFileDialog.ShowDirsOnly)
 
+      if not self.CurrentFolder:
```

Comment:
Is this related to stop the app crashing if we do not select a folder after having clicked on browse volume folder? maybe a comment in the code could be nice!

Reply:
yes! added a comment